MomentoIntegrationTest/SimpleCacheDataTest.cs

```diff
@@ -17,6 +17,7 @@ public SimpleCacheDataTest(SimpleCacheClientFixture fixture)
     {
         client = fixture.Client;
         authToken = fixture.AuthToken;
+	cacheName = fixture.CacheName;
```

Comment:
Ditto

Reply:
Fixed in [02e561e](https://github.com/momentohq/client-sdk-csharp/pull/105/commits/02e561ebfd9e54cff79b3ecc665475cb41c3f7db)